src/apis/productApi.ts

```diff
@@ -0,0 +1,39 @@
+import type {
+  ProductBasicInfo,
+  ProductDetailInfo,
+  ProductHighlightReview,
+  ProductId,
+  ProductWishInfo,
+} from 'src/types/product';
+import axiosInstance from './axiosInstance';
+
+// 상품 기본정보 API
+export const fetchProduct = async (
+  id: ProductId
+): Promise<ProductBasicInfo> => {
+  const res = await axiosInstance.get(`/products/${id}`);
```

Comment:
[제안] 반환 타입을 axios에 제네릭으로 전달하면 타입 추론이 데이터의 출처부터 타입추론이 더 잘될 듯 하네요
```suggestion
) => {
  const res = await axiosInstance.get<ProductBasicInfo>(`/products/${id}`);
```